### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by vchuravy** [FedeClaudi/Term.jl#105] [Ideas]
I was wondering if you had any thoughts on how to implement pagination?

In particular I would like to build TUI's and ideally have them be reactive to the size of the terminal.

**Comment by FedeClaudi** [comment]
Hi, thanks for your interest.

The short answer is that it will happen, but not soon. 

The longer answer is that the next release, version `1.0` introduces a new object called `Compositor` which starts building the infrastructure to create terminal visualizations that can be easily updated. There's a few steps that need to happen before this can turn into something that can be used to make an actual TUI, but more reactive/interactive widgets will start coming out gradually. 

It will likely start being one of the priorities after the release of v1.0 in a couple of weeks.
